piton/tools/src/proto/common/setup.tcl

```diff
@@ -97,10 +104,17 @@ if {[info exists ::env(PITON_ARIANE)]} {
   append ALL_DEFAULT_VERILOG_MACROS " PITON_ARIANE PITON_RV64_PLATFORM PITON_RV64_DEBUGUNIT PITON_RV64_CLINT PITON_RV64_PLIC WT_DCACHE"
 }
 
+if {[info exists ::env(PITON_BLACKPARROT)]} {
+  append ALL_DEFAULT_VERILOG_MACROS " PITON_BLACKPARROT PITON_RV64_PLATFORM PITON_RV64_DEBUGUNIT PITON_RV64_CLINT PITON_RV64_PLIC"
```

Comment:
I think we probably don't want PITON_RV64_DEBUGUNIT here right?

Reply:
Yeah probably. We do have debug unit support, but I think it's not hooked up right now